src/clj/rems/api/schema.clj

```diff
@@ -98,6 +98,7 @@
    (s/optional-key :decider) s/Str
    (s/optional-key :decision) s/Keyword
    (s/optional-key :commenters) #{s/Str}
+   (s/optional-key :latest-comment-request-by-user) {s/Str s/Uuid}
```

Comment:
This is only used by the commands, so it should not be exposed to the UI. Please remove it from the public schema.

It might be good to also change it to a private namespaced keyword, i.e. `::latest-comment-request-by-user`, so that it would be tied to the `rems.workflow.dynamic` namespace and not accidentally be used elsewhere.